doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -97,7 +97,7 @@ The following list summarizes the most important changes inherited from the upst
 Thread
 ------
 
-|no_changes_yet_note|
+* Initial experimental support for CLI and coprosessor samples for nRF54l15.
```

Comment:
```suggestion
* Initial experimental support for nRF54L15 for Thread CLI and Co-processor samples.
```
Same information needs to be propagated in the Sample section for Thread samples, similarly to https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/releases_and_maturity/releases/release-notes-changelog.html#nfc-samples .

Reply:
fixed.